src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -389,7 +389,11 @@ public function getGroupedFeatures()
     public function getSeoAvailability()
     {
         $seoAvailability = 'https://schema.org/';
-        if ($this->product['quantity'] > 0) {
+
+        // Availability for displaying discontinued products, if enabled
+        if ($this->product['active'] != 1) {
+            $seoAvailability .= 'Discontinued';
```

Comment:
Also just a thought (not required), but shouldn't it be better to put these values into some constants (at least here in the ProductLazyArray if they are not used anywhere else for now)? Its not recommended having plain strings hanging.

Reply:
src/Core/Domain/Product/ValueObject/SeoAvailability?

Reply:
hmm that wouldn't fit into a ValueObject definition, because its not something we will initialize, it will only hold the constants. (ValueObjects should be initializable and validate itself  - so a good example of value object would be if we need something like `new SeoAvailability($value)`. But in this case we won't ever initialize the class. I don't have any other suggestions where to put it though... Maybe @jolelievre have any thoughts?

Reply:
Guys is this really needed? It's one place where this is used. Do we need to test that if I create new SeoAvailability it will return the string I hardcoded there? ... come on

Reply:
@Hlavtox it's easier to maintain this way, if they ever change Available to "SomethingElse" you change the value in the constant and the rest of the system works as before

@zuk3975 I edited my message right after I wrote about VO ;)

Reply:
@kpodemski Yeah probably sounds good. Though it would need comments above the class about what it represents (and a private constructor, so it cannot be initialized), so we don't confuse it with a value object.

Reply:
I have other PR that modifies this, so let's do this after it's merged pls ;-) https://github.com/PrestaShop/PrestaShop/pull/29570